### ISSUE TITLE
Documentation suggestion: mention that G502 support requires using onboard profiles

### ISSUE THREAD

**Issue by spencerwi** [libratbag/libratbag#1286]
**Is your feature request related to a problem? Please describe.**
I use my G502 mouse and random wired keyboard with two computers, by way of a nice USB device switcher: my linux workstation for work, and my Windows personal machine for gaming. I tried to use Logitech's G Hub software on the Windows side to configure my mouse there, and Piper on the Linux side to configure my mouse for Linux. 

I ran into weird issues where Piper worked fine, until I installed and tried to use G Hub on Windows. After that point, Piper would happily claim to have applied changes, but nothing seemed to actually take effect.

After much tinkering. I realized that what ratbagctl and Piper are both doing is modifying the mouse's onboard profiles -- but since in the G Hub software on the Windows side I had swapped the mouse _away_ from using onboard profiles, none of my changes were actually in use.  

**Describe the solution you'd like**
It'd be helpful to document that for the Logitech G502 series (and possibly others?), what Ratbag does is modify _onboard device profiles_, and so in order for changes to take effect, your mouse _must_ be using onboard profiles.

**Describe alternatives you've considered**
Ideally, it'd be nice if libratbag and friends worked with non-onboard profiles so that I could configure my mouse differently between Windows and Linux, but that seems like a much harder goal to achieve, so a bit of extra documentation seems more plausible.  

**Additional context**
N/A

**Comment by amfern** [comment]
You are right, it's not straight forward that we use only the onboard profiles.
Also, I noticed a bug, libratabg suppose to switch the mouse to use the onboard profile on connection #1303

**Comment by lombervid** [comment]
I would really like it to work with non-onboard profiles, since I still use Windows. And, in order to use automatic profile switching on Windows, onboard profiles must be disabled.

Besides that, is there a way to enable onboard profiles with `ratbagctl` command?
